fhir-server/pom.xml

```diff
@@ -135,6 +135,21 @@
             <artifactId>jakarta.enterprise.cdi-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jacorb</groupId>
+                    <artifactId>jacorb-omgapi</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.spec.javax.rmi</groupId>
+                    <artifactId>jboss-rmi-api_1.0_spec</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
can we remove this altogether now?  if not, why not?

Reply:
I'll paste error I was getting when I tried to removed it.

Reply:
[ERROR] testBuildCapabilityStatement_resources_empty(com.ibm.fhir.server.test.CapabilitiesTest)  Time elapsed: 4.913 s  <<< FAILURE!
java.lang.ExceptionInInitializerError
	at com.ibm.fhir.server.test.CapabilitiesTest.testBuildCapabilityStatement_resources_empty(CapabilitiesTest.java:63)
Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
	at com.ibm.fhir.server.test.CapabilitiesTest.testBuildCapabilityStatement_resources_empty(CapabilitiesTest.java:63)
Caused by: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
	at com.ibm.fhir.server.test.CapabilitiesTest.testBuildCapabilityStatement_resources_empty(CapabilitiesTest.java:63)

Reply:
Same answer as operation/fhir-operation-bulkdata/pom.xml.  This 'provided' dependency needs to remain for now.

Reply:
I'm confused.  That looks like a client-side error to me.  
I'm not sure why having a provided dependency in fhir-server matters for the client error.  Did you try changing the dependency scope to `test`?

Reply:
I'll try that.

Reply:
Changing it to 'test' worked.